x-pack/plugins/ml/server/routes/schemas/inference_schema.ts

```diff
@@ -24,6 +24,10 @@ export const modelAndDeploymentIdSchema = schema.object({
    */
   deploymentId: schema.string(),
 });
+export const createInferenceSchema = schema.object({
+  taskType: schema.string(),
```

Comment:
```suggestion
  taskType: schema.oneOf([
    schema.literal('sparse_embedding'),
    schema.literal('text_embedding'),
  ]),
```

Reply:
added in https://github.com/elastic/kibana/pull/180330/commits/babdac0c379e95518694281a95eede8f4a06cc3d